src/main/java/edu/usf/cutr/tba/utils/StringUtils.java

```diff
@@ -61,15 +61,15 @@ public static long validateStringDateAndParseToMillis(String dateStr) {
      * @return String of a valid path or "" if the path is invalid or if its not possible to
      * be created.
      */
-    public static String validateAndParseOutputPath(String newPath) {
+    public static String validateAndParseFolderPath(String newPath) {
         try {
             Path localPath = Paths.get(newPath);
+
             if (Files.exists(localPath)) {
                 return localPath.toString();
             } else {
                 // Folder does not exists, try to create it
-                System.out.println("The provided path does not exist. " +
-                        "Trying to create the folder: '" + localPath.toString() + "' ...");
+                System.out.println("Creating new folder: '" + localPath.toString() + "' ...");
```

Comment:
I did the change in the most recent PR. 
The userId's are still visible from the folder names. Show we look for a different naming convention for folders?

Reply:
Thanks! No, I think the folder names are fine because they are on disk in the same place the sensitive data is being stored so they can be assumed as being protected. But the system logs could end up a variety of places, especially of standard out is redirected, and we can't guarantee that the location is protected.